Tools/Scripts/libraries/webkitbugspy/webkitbugspy/bugzilla.py

```diff
@@ -447,3 +466,52 @@ def create(
             )
             return None
         return self.issue(response.json()['id'])
+
+    def cc_radar(self, issue, block=False, timeout=None):
+        if not self.radar_importer:
+            sys.stderr.write('No radar importer specified\n')
+            return None
+
+        for tracker in Tracker._trackers:
+            if isinstance(tracker, RadarTracker):
+                break
+        else:
+            sys.stderr.write('Project does not define radar tracker\n')
+            return None
+
+        did_modify_cc = False
+        if self.radar_importer not in issue.watchers:
+            log.info('CCing {}'.format(self.radar_importer.name))
```

Comment:
Could you explain why we want to mix usages between `sys.stderr` and `log`?

Reply:
`git-webkit` respects `-v` and `-q`. I try and reserve `sys.stderr` for errors, and `log` for informational messages. Put another way, you can silence `log` messages (or they may be silenced by default, as is the case with `log.info`) where as you cannot silence `sys.stderr` or `sys.stdout` with command line arguments.